src/dialog-user/components/dialog-user/dialogField.ts

```diff
@@ -127,6 +121,10 @@ export class DialogFieldController extends DialogFieldClass {
   public refreshSingleField() {
     this.singleRefresh({ field: this.field.name });
   }
+
+  public fieldTokens(element) {
+    return element.innerText + ' ' + element.value.split(/:/)[1];
```

Comment:
Seems weird to rely on the html, we should have the same thing that `value[0]` and `value[1]` were in `data-tokens` from `this.field.values[0]` and `this.field.values[1]`, right? Or am I misunderstanding what this is doing?

Reply:
You're exactly right, I'll try. 
But I haven't found any scope where this could be read from, `angular.element(optionElement).scope().value` gives me undefined.

So we'd have to manually match option elements to entries in the array, to be able to do that, which may be a bit tricky to do right around the empty option, but probably doable :)

also, TODO, `$watchCollection`

Reply:
Heheh, so, I got this working with `miq-options="{ 'data-tokens': value[0] + ' ' + value[1] }"` :).
But for now not dealing with the empty option at all, simply..

```js
      var match = attrs.ngOptions.match(/for\s+(.+)\s+in\s+(.+)\s*/);
      if (!match) {
        return;
      }
      var attrName = match[1];
      var attrToWatch = match[2];

      var updateOptions = function (newValue) {
        if (!newValue) {
          return;
        }

        angular.element('option', element).each(function (i, e) {
          var fakeScope = {
            $parent: scope,
            $index: i,
          };
          fakeScope[attrName] = newValue[i];                  // <---

          var miqOptions = $parse(attrs.miqOptions)(fakeScope);
          Object.keys(miqOptions).forEach(function(k) {
            var v = miqOptions[k];
            if (_.isFunction(v)) {
              v = v(e);
            }

            angular.element(e).attr(k, v);
          });
        });
      };

      scope.$watch(attrToWatch, updateOptions);
      scope.$watchCollection(attrToWatch, updateOptions);
```

Reply:
Pushed :)